### PR TITLE
Hack

### PR DESCRIPTION
**Pull request by vuplea**

### CODE REVIEW

File: .gdbinit [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Source all settings from the peda dir
+source ~/gits/peda/peda.py
```
Comment (vuplea) [inline .gdbinit]: why not reference it directly?

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	fi
+
+	set +x
+}
+
+install_pgdb() {
+	set -x
+
+	mkdir -p ~/gits
+	ln -fs `realpath ./gits/peda` ~/gits &> /dev/null
```
Comment (vuplea) [inline install.sh]: todo

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+	rm -rf ~/.vim
+	ln -fs `realpath ./.vim` ~
+	ln -fs `realpath ./.vimrc` ~
+	ln -fs `realpath ./.gvimrc` ~
+
+	if command -v nvim &> /dev/null; then
+		mkdir -p ~/.scripts
+		rm -rf ~/.scripts/nvim
+		ln -fs `realpath ./.scripts/nvim` ~/.scripts/
```
Comment (vuplea) [inline install.sh]: todo

File: .vimrc [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+" we still keep the functionality of the semicolon.
+noremap ; :
+noremap : ;
+
+" Quick replay 'q' macro and avoid Ex-mode.
+noremap Q <nop>
+nnoremap Q @q
+
+" Reuse x as a proper delete.
+noremap x "_d
```
Comment (mdumitru) [inline .vimrc]: xl. But x is useless.

File: .vimrc [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+nnoremap <silent> <f3> :ToggleColorColumn<cr>
+
+
+"------ Global shortcuts ------
+" Uniform mappings that can be used from neovim's terminal.
+
+" Make it easy to close stuff (remember that 'hidden' is set).
+noremap <silent> <a-q> :q<cr>
+noremap <silent> <a-backspace> :Bdelete<cr>
+noremap <silent> <leader><backspace> :bdelete<cr>
```
Comment (mdumitru) [inline .vimrc]: Close the split.

**Pull request closed without merging**